app/javascript/dashboard/modules/widget-preview/components/WidgetFooter.vue

```diff
@@ -1,38 +1,101 @@
 <template>
   <div class="footer-wrap">
-    <div class="input-area" />
+    <custom-button
+      v-if="config.isDefaultScreen"
+      class="start-conversation"
+      :style="{ background: config.color }"
+    >
+      {{
+        $t('INBOX_MGMT.WIDGET_BUILDER.FOOTER.START_CONVERSATION_BUTTON_TEXT')
+      }}
+    </custom-button>
+    <div v-else class="chat-message--input is-focused">
+      <resizable-text-area
+        id="chat-input"
+        :placeholder="
+          $t('INBOX_MGMT.WIDGET_BUILDER.FOOTER.CHAT_INPUT_PLACEHOLDER')
+        "
+        class="user-message-input is-focused"
+      />
+      <div class="button-wrap">
+        <fluent-icon icon="emoji" />
+        <fluent-icon class="icon-send" icon="send" />
+      </div>
+    </div>
   </div>
 </template>
 
 <script>
+import CustomButton from '../../../../dashboard/components/buttons/Button';
+import ResizableTextArea from '../../../../shared/components/ResizableTextArea';
 export default {
   name: 'WidgetFooter',
+  components: {
+    CustomButton,
+    ResizableTextArea,
+  },
+  props: {
+    config: {
+      type: Object,
+      default: () => {},
+    },
+  },
 };
 </script>
 
 <style scoped lang="scss">
+@import '~dashboard/assets/scss/variables.scss';
 .footer-wrap {
-  flex-shrink: 0;
   width: 100%;
   display: flex;
   flex-direction: column;
   position: relative;
-  &::before {
-    content: '';
-    position: absolute;
-    top: var(--space-one);
-    left: 0;
-    width: 100%;
-    height: var(--space-one);
-    opacity: 0.1;
-    background: var(--color-background);
+  padding-left: var(--space-two);
+  padding-right: var(--space-two);
+
+  .start-conversation {
+    justify-content: center;
+    font-size: var(--font-size-small);
+    border-radius: var(--border-radius-normal);
+    padding-left: var(--space-two);
+    padding-right: var(--space-two);
   }
 
-  .input-area {
-    background-color: var(--white);
+  .chat-message--input {
+    align-items: center;
+    display: flex;
+    padding: 0 var(--space-small) 0 var(--space-slab);
     border-radius: var(--border-radius-normal);
-    height: var(--space-larger);
-    margin: var(--space-two);
+    background: white;
+
+    &.is-focused {
+      box-shadow: 0 0 0 1px $color-woot, 0 0 2px 3px $color-primary-light;
```

Comment:
Please use css variables instead of sass variables, as we are not using these anymore

Reply:
Done